### ISSUE TITLE
jpa2ddl : Workspace build error in Eclipse

### ISSUE THREAD

**Issue by mo-costello** [Devskiller/hbm2ddl-maven-plugin#30]
Greetings,

I have just integrated the jpa2ddl maven plugin to automate the scripts generation with Hibernate 5.2.3 and the database migration with Flyway.

While the scripts generation/execution works fine when running a maven build on the project in Eclipse. The plugin integration has caused the following error after every file save or workspace build : 

```plain
Errors occurred during the build.
Errors running builder 'Maven Project Builder' on project 'emap-lib'.
Missing parameter for pluginExecutionFilter. groupId, artifactId, versionRange and goals must be specificed, but found: groupId = 'com.devskiller.jpa2ddl'
artifactId = 'jpa2ddl-maven-plugin'
versionRange = '0.9.12'
goals = '[]'
```

Below is the relevant pom section : 

```xml
<plugin>
    <groupId>com.devskiller.jpa2ddl</groupId>
    <artifactId>jpa2ddl-maven-plugin</artifactId>
    <extensions>true</extensions>
    <configuration>
        <packages>
            <package>com.sample.entities</package>
        </packages>
        <formatOutput>true</formatOutput>
        <delimiter>;</delimiter>
        <outputPath>${basedir}/src/main/resources/db/migration/</outputPath>
        <action>UPDATE</action>
    </configuration>
</plugin>
```

Even when specifying the goal "generate", Eclipse Maven Project Builder will still fail.

Would you please advise ?
Thank you and best regards.

**Comment by m-pavel** [comment]
Plugin jar contains invalid lifecycle-mapping-metadata.xml (META-INF/m2e/lifecycle-mapping-metadata.xml) with empty 'goals' property. As 'dirty' fix you can remove this file from plugin jar .m2/repository/com/devskiller/jpa2ddl/jpa2ddl-maven-plugin/0.9.12/jpa2ddl-maven-plugin-0.9.12.jar. As valid fix takari-lifecycle-plugin need to be updated.

**Comment by jkubrynski** [comment]
Update will be deployed with the next release

**Comment by jkubrynski** [comment]
Available in version 0.10.0

**Comment by vitalivu992** [comment]
Still found it in 0.10.0 `The parameters 'packages' for goal com.devskiller.jpa2ddl:jpa2ddl-maven-plugin:0.10.0:generate are missing or invalid`
